tensor/dense_io_test.go

```diff
@@ -27,7 +27,7 @@ func TestSaveLoadNumpy(t *testing.T) {
 
 	script := "import numpy as np\nx = np.load('test.npy')\nprint(x)"
 
-	cmd := exec.Command("python")
+	cmd := exec.Command("python2")
```

Comment:
Should be. As far as I could tell this was the most efficient  and universal way to do it - simply setting masked values to the fill value means that arrays can be opened in/from numpy. If you want to save the raw mask too, then I figured we could just leave that to the programmer to implement in whatever format they prefer.